### PR TITLE
added eslint

### PR DESCRIPTION
**Pull request by burymm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,13 @@
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
     "test": "ng test",
-    "server": "json-server --watch ./server/database.json"
+    "server": "json-server --watch ./server/database.json",
+    "lint": "ng lint"
+  },
+  "husky": {
```
Comment (Dewlar) [inline package.json]: husky dependency in devDependencies was missed.
use `npm install husky --save-dev` command.
like this
Screenshot_1

**Review by Dewlar — COMMENTED** [review]
`husky` and `lint-staged` dependency in `devDependencies` was missed.

File: package.json [hunk 2]
```diff
@@ -10,9 +10,16 @@
...
+  "lint-staged": {
+    "src/**/*.{js,ts,scss,md,html,json}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
   "husky": {
     "hooks": {
-      "pre-commit": "lint"
+      "pre-commit": "ng lint && lint-staged",
```
Comment (Dewlar) [inline package.json:21]: `lint-staged` is specified which is also omitted from `devDependencies` of `package.json` file
you must use the `--save-dev` flag

File: package.json [hunk 3]
```diff
@@ -10,9 +10,16 @@
     "server": "json-server --watch ./server/database.json",
     "lint": "ng lint"
   },
+  "lint-staged": {
```
Comment (Dewlar) [inline package.json]: `lint-staged` is listed but also omitted from `devDependencies` in the `package.json` file
use `npm i lint-staged --save-dev` command

**Comment by burymm** [comment]
closed as duplicate

**Pull request closed without merging**